.github/workflows/deploy-review-command.yml

```diff
@@ -63,6 +63,31 @@ jobs:
               "GITHUB_TRIGGER": "${{ github.event.client_payload.github.payload.comment.html_url }}"
             }
 
+      - name: Check status
```

Comment:
Could we potentially encounter a situation where this task starts before the GitLab pipeline?

Reply:
no chance